net/suricata/files/suricata.init

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh /etc/rc.common
+# (C) 2018 Ashkan Jazayeri <ashkan@jazayeri.net>
+
+START=75
+USE_PROCD=1
+
+validate_suricata_section() {
+	uci_validate_section suricata suricata "${1}" \
+		'mode:string:af-packet' \
+		'queue:list(range(0,65535))' \
+		'verbose:range(0,4):0'
+}
+
+start_service() {
+	local mode number queue verbose
+	validate_suricata_section instance || {
+		echo "validation failed"
+		return 1
+	}
+	[ -d /var/log/suricata ] || mkdir /var/log/suricata
+	[ -f /var/run/suricata.pid -a -z $(pgrep suricata) ] && rm /var/run/suricata.pid && logger -t suricata[init_script] -p daemon.alert -s "Suricata was not closed properly or it has crashed. Successfully removed the previous /var/run/suricata.pid"
+	procd_open_instance
+	procd_set_param command /usr/bin/suricata
+    [ "$verbose" -gt 0 ] && {
+		procd_append_param command -$(printf 'v%.0s' $(seq 1 $verbose))
+		procd_set_param stdout 1
+		procd_set_param stderr 1
+	}
+	case "$mode" in
+		"af-packet" ) procd_append_param command --af-packet
+		;;
+		"nfq" )
+			[ -n "$queue" ] || {
+				logger -t suricata[init_script] -p daemon.emerg -s "No queue list provided. In NFQUEUE mode, a queue list must be specified under suricata config section (e.g. uci add_list suricata.instance.queue=9)"
+				return 1
```

Comment:
why logger instead of printf/echo?  this will silently fail to a user.

Reply:
from logger man: 
> -s log the message to standard error, as well as the system log.